scripts/issue265-gh-shim.sh

```
@@ -0,0 +1,290 @@
+    return 0
+}
+
+# ─── Exact production argument vectors ───────────────────────────────────
+#
+# These are extracted verbatim from the production source code for the
+# issue #265 scenario (owner=owner, repo=repo-265, number=265, filter=default
+# open, page_size=30, no cursor).
+
+# The exact GraphQL search-list query body (no cursor — first page).
+# Source: src/github/parse.rs build_issue_search_args, cursor=None branch.
+readonly SEARCH_QUERY_BODY='query($searchQuery: String!, $first: Int!) { search(type: ISSUE, query: $searchQuery, first: $first) { nodes { ... on Issue { id number title state author { login } updatedAt assignees(first: 10) { nodes { login } } labels(first: 20) { nodes { name } } issueType { name } milestone { title } comments { totalCount } } } pageInfo { hasNextPage endCursor } } }'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)